tmt/steps/execute/internal.py

```diff
@@ -16,42 +16,39 @@
     "TMT_REBOOT_COUNT",
     "REBOOTCOUNT",
     "RSTRNT_REBOOTCOUNT")
+bin_dir = "/usr/local/bin/"
```

Comment:
Make this constant an upper-case, please, for consistency :)

Reply:
Yep, that makes sense.